packages/cms/src/utils/isIE.js

```diff
@@ -0,0 +1,16 @@
+/** check if the current browser is internet explorer */
+function isIE() {
+  let isIE = false;
+
+  // make sure the browser window exists
+  if (typeof window !== "undefined") {
+    // will only return true for internet explorer
+    if (/*@cc_on!@*/false || !!document.documentMode) { // eslint-disable-line spaced-comment
+      isIE = true;
+    }
+  }
+
+  return isIE;
+}
+
+module.exports = isIE;
```

Comment:
Ahh, I see you didn't test this in non-IE browsers 😉 

This file exports a _Function_, which based on the way that you reference it in all of your components (`isIE`), it will always return `true` because a JavaScript _Function_ is a "truthy" variable. In it's current form, you would have to invoke this function to actually get the value (`isIE()`), but that would be running this code on every single render in every single usage. Ideally, this function code just be the following, but please test this in IE and Chrome to confirm:

```js
/** check if the current browser is internet explorer */
let isIE = false;

// make sure the browser window exists
if (typeof window !== "undefined") {
  // will only return true for internet explorer
  if (/*@cc_on!@*/false || !!document.documentMode) { // eslint-disable-line spaced-comment
    isIE = true;
  }
}

module.exports = isIE;
```